packages/origin-backend/migrations/1596616659110-OrganizationInvitationSenderEmail.ts

```diff
@@ -0,0 +1,27 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class OrganizationInvitationSenderAndProcessed1596616659110 implements MigrationInterface {
+    name = 'OrganizationInvitationSenderAndProcessed1596616659110';
+
+    public async up(queryRunner: QueryRunner): Promise<any> {
+        await queryRunner.query(
+            `ALTER TABLE "organization_invitation" ADD "sender" character varying NOT NULL`,
```

Comment:
Adding NOT NULL columns require a default value or will not migrate already existing DB records and fail the process.